vite.config.js

```diff
@@ -1,5 +1,6 @@
 import { defineConfig } from 'vite';
 import laravel from 'laravel-vite-plugin';
+import path from 'path'
 
 export default defineConfig({
     plugins: [
```

Comment:
could do with adding the laravel plugin here

```
laravel([
     'resources/sass/app.scss', 
     'resources/js/app.js',
])
 ```

Reply:
@dcblogdev, could you please provide more clarification?